test/e2e/replica_set/replica_set_test.go

```diff
@@ -32,6 +32,30 @@ func TestReplicaSet(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	t.Setenv("MDB_RUN_AS_TEST", "true")
+
+	sizeThresholdMB := automationconfig.StringAsFloat("0.0001")
```

Comment:
Are fractions supported here? Also leave a comment that it's 100 bytes or change var name to indicate that, e.g. sizeThreshold100BytesInMB.

Reply:
Wouldn't it create hundreds of rotated files in a very short time if executed in e2e test?
Can exec into the pod and check if the files were rotated and are about the size we expect here? If we don't have anything like already in tests then don't spend time on it.